doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -273,14 +273,23 @@ Matter samples
   * The :file:`prj_no_dfu.conf` file.
   * Support for ``no_dfu`` build type for nRF5350 DK, nRF52840 DK and nRF7002 DK.
 
-* Added test event triggers to all Matter samples.
-  By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.
+* Added:
+
+  * Test event triggers to all Matter samples.
+    By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.
+
+    To get started with using test event triggers in your Matter samples and to understand the capabilities of this feature, refer to the :ref:`ug_matter_test_event_triggers` page.
+
+  * Support for the nRF54L15 PDK with the ``nrf54l15pdk_nrf54l15_cpuapp`` board target to the following Matter samples:
```

Comment:
```suggestion
  * Support for the nRF54L15 PDK with the ``nrf54l15pdk_nrf54l15_cpuapp`` build target to the following Matter samples:
```
Does the build target format need to change to align with PR#14626?

Reply:
right, it should be "configuration" and "suffix" instead of build target :)

Reply:
or wait, this is a build target, so it is not a suffix of the prj.conf file.

Reply:
I was just wondering because, as I understand the other PR, build target names will change from using underscores to using slashes.